lib/googleauth/user_authorizer.rb

```diff
@@ -57,14 +58,19 @@ class UserAuthorizer
       # @param [String] callback_uri
       #  URL (either absolute or relative) of the auth callback.
       #  Defaults to '/oauth2callback'
-      def initialize client_id, scope, token_store, callback_uri = nil
+      # @param [String] code_verifier
+      #  Random string of 43-128 chars used to verify the key exchange using
+      #  PKCE. Auto-generated if not provided
```

Comment:
The code doesn't look like it "auto-generates" the verifier if not provided. It looks like it simply doesn't include the code challenge.

Reply:
Ah yes! That was the implementation I was trying to follow but it doesn't work given the way our 3LO code is implemented. Removed that.